docs/navbar/index.html

```diff
@@ -86,6 +86,44 @@ <h2 class="i-logo">
         </aside>
         <article>
           <h3>Navbar</h3>
+          <div class="component-description">
+            <p class="justify">
+              Navbar is sometimes the very first component that gets created in
+              a website. Looks simple from outside, but is time consuming in
+              creating. <span>Chunks</span> provides simple navbar which user
+              can use as described below.
+            </p>
+          </div>
+          <h4>Simple Navbar</h4>
+          <div class="component-example">
+            <nav class="navbar">
+              <div class="nav-logo">Logo</div>
+              <div class="nav-list">
+                <ul class="i-nav-item">
+                  Home
+                </ul>
+                <ul class="nav-item">
+                  About
+                </ul>
+                <ul class="nav-item">
+                  Blogs
+                </ul>
+              </div>
+            </nav>
+          </div>
+
+          <div class="code-example">
+            <pre><code class="html">
```

Comment:
is it a good practice to name class as html element?

Reply:
thats a requirement of highlightjs